packages/calypso-products/src/main.ts

```diff
@@ -75,11 +93,38 @@ export function getPlans(): Record< string, Plan > {
 }
 
 export function getPlanFeaturesGrouped(): Partial< FeatureGroupMap > {
-	return featureGroups;
+	if ( isTrailMapStructureVariant() ) {
```

Comment:
```suggestion
	if ( isTrailMapCopyVariant() ) {
```

^ IMO, since the new feature groups should be shown in the copy-only variant.